tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/resolver/TestMetadataInputHouseResolver.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.metadata.extension.resolver;
+
+import static org.mule.test.metadata.extension.resolver.TestMetadataResolverUtils.getHouseMetadata;
+
+import org.mule.metadata.api.model.MetadataType;
+import org.mule.runtime.api.connection.ConnectionException;
+import org.mule.runtime.api.metadata.MetadataContext;
+import org.mule.runtime.api.metadata.MetadataResolvingException;
+import org.mule.runtime.api.metadata.resolving.InputTypeResolver;
+
+public class TestMetadataInputHouseResolver implements InputTypeResolver<String> {
+
+  public static String TEST_INPUT_HOUSE_RESOLVER = "testInputHouseResolver";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static TEST_INPUT_HOUSE_RESOLVER" field final ([squid:S1444](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1444))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7668&issues=AWjdfEJ_otIXnmsUGfbM&open=AWjdfEJ_otIXnmsUGfbM)

[ISSUE_KEY]: AWjdfEJ_otIXnmsUGfbM